R/plugin-awesomeMarkers.R

```diff
@@ -0,0 +1,187 @@
+leafletAwesomeMarkersDependencies <- function() {
+  list(
+    htmltools::htmlDependency(
+      "leaflet-awesomemarkers",
+      "2.0.2",
+      system.file("htmlwidgets/plugins/Leaflet.awesome-markers", package = "leaflet"),
+      script = c('leaflet.awesome-markers.min.js','bootstrap.min.js'),
+      stylesheet = c('leaflet.awesome-markers.css','bootstrap.min.css',
```

Comment:
I can split them out, I even thought of doing that. But then it would mean I can't add them by default , right ? The dependency will have to be added by an extra function call , something like addBootStrap() / addFontAwesome() , right ?